File: 22.BMI_calculator/python/BMI_calculator.py
import re
print ('Please eneter your:')
weight = input('weight in kilograms: ')
height = input('height in centimiters: ')
match = re.match('[a-zA-Z]*$',weight)
if match:
    print ('Numbers only allowed')
    weight = input('weight in kilograms: ')
weight = int(weight)
height = int(height)
BMI = weight / ((height/100)**2)
print (BMI)

File: 22.BMI_calculator/php/BMI_calculator.php
<?php
session_start();
$weight = (int) $_GET['weight'];
$height = (int) $_GET['height'];

function validateInput($input)
{

}

function calculateBMI(int $weight, int $height):float
{
 $bmi = $weight / (($height/100)^2);
 return $bmi;
}


$_SESSION['bmi'] = calculateBMI($weight,$height);
header('Location: index.php');
exit();
?>
